lib/l10n/intl_es.arb

```
@@ -610,6 +610,7 @@
   "disputeTradeDialogContent": "Estás a punto de iniciar una disputa con tu contraparte. ¿Deseas continuar?",
   "disputeCreatedSuccessfully": "Disputa creada exitosamente",
   "disputeCreationFailed": "Falló la creación de la disputa",
+  "disputeCreationError": "Error al crear la disputa",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)